grid/social-media/followers/profile.supr

```diff
@@ -0,0 +1,95 @@
+name = "social-media/followers"
+version = "1.0.0"
+
+"""
+Get Followers
+Retrieve paginated list of social-media profile followers.
+"""
+usecase GetFollowers safe {
+  input {
+    """
+    Profile ID
+    Identification of profile to retrieve posts from; may be optional with some providers.
+    """
+    profileId string
+    """
+    Page
+    Token of the page to retrieve
+    """
+    page string
+  }
+
+  result {
+    """
+    Previous page
+    Previous page token
+    """
+    previousPage string
+    """
+    Next page
+    Next page token
+    """
+    nextPage string
+    """
+    Followers
+    List of followers.
+    """
+    followers [ {
+      """
+      Follower ID
+      Unique follower identification.
+      """
+      id string
+      """
+      Username
+      Username of the follower.
+      """
+      username string
+      """
+      Image url
+      Follower profile image url.
+      """
+      imageUrl string
+    } ]
+  }
+
+  error {
```

Comment:
I suggest there should be an error model suitable for rate limiting which a) is easy to capture b) includes details about the limit.

Reply:
What about new property `retryAfter`?

Reply:
we can get that info `retryAfter` from headers 👍, I will add it in separate PR